app/models/loader/types.rb

```diff
@@ -201,6 +201,12 @@ class User < Record
 
       # Below is a sample method verifying policy data validity
       def verify
+        user_creation_allowed = ENV['CONJUR_ALLOW_USER_CREATION']
```

Comment:
might want to change this to something like:

```suggestion
        user_creation_allowed = ENV.fetch('CONJUR_ALLOW_USER_CREATION', 'true').downcase
```
to allow variations of false below.

Reply:
done